### ISSUE TITLE
Integrate phosphor markers

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#188]
## User story
1. As a User
2. I want different markers (icons)
3. So that I can differantiate between different POIs / categegories for markes on the map

## Decisions
* Use phosphor markers 

## Acceptance criteria
* Integrate open source phosphor markers to display beautiful icons
* Documentation
* Possibility to merge icons to create new ones

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Lucas-Nan** [comment]
markers are usually required as png directly imported. Instead now can use all icons for example from phosphor icons or material ui
